centrifuge-app/src/utils/formatting.ts

```diff
@@ -48,3 +48,7 @@ export function formatPercentage(amount: Perquintill | Decimal | number, include
   })
   return includeSymbol ? `${formattedAmount}%` : formattedAmount
 }
+
+export function roundDown(float: Decimal | number) {
+  return Math.floor((float instanceof Decimal ? float.toNumber() : float) * 100) / 100
+}
```

Comment:
This should probably accept a precision. Now it's implicitly rounding to a precision of 2 decimals. Something like:

```suggestion
export function roundDown(float: Decimal | number, precision: number = 2) {
  return Math.floor((float instanceof Decimal ? float.toNumber() : float) * 10**precision) / 10**precision
}
```